src/stores/Archaeologist/index.tsx

```diff
@@ -33,8 +33,8 @@ const createDataRoot = () => {
 
 const ArchaeologistsProvider = createDataRoot();
 
-const useArchaeologistsStore = () => {
-  return useContext(context);
+const useArchaeologistsStore = (ctxt?: any) => {
```

Comment:
this was the only way I could figure out how to test context. all ears to change.